Project.toml

```diff
@@ -1,11 +1,14 @@
 name = "JavisNB"
 uuid = "92afb270-2599-44f6-96a1-44c6efb1daf1"
 authors = ["Jacob Zelko <jacobszelko@gmail.com>", "Giovanni Puccetti <giovanni.puccetti@sns.it>"]
-version = "0.7.0"
+version = "0.1.0"
 
 [deps]
 IJulia = "7073ff75-c697-5162-941a-fcdaad2a7d2a"
+Interact = "c601a237-2ae4-5e1e-952c-7a85b0c7eef1"
+Javis = "78b212ba-a7f9-42d4-b726-60726080707e"
 Pluto = "c3e4b0f8-55cb-11ea-2926-15256bba5781"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
 
 [compat]
-julia = "1.5"
+julia = "1.7"
```

Comment:
We want to allow 1.6 as it will be the new LTS

Reply:
Will fix!